### DISCUSSION TITLE
Migration from v5 to v6?

### DISCUSSION THREAD

**Discussion by CrazyKidJack** [nodejs/undici#3076] [Q&A] [answered]
Hello all, sorry if I missed it somewhere. Is there any documentation explaining the differences (specifically any breaking changes) in the API or behavior between v5 and v6? Or perhaps a migration guide?

There is another project that provides a template repo that I cloned. Dependabot wants me to update from 5.27.2 to 5.28.4 for security. However, while looking into that, I also noticed undici has released v6. If there is any migration guidance I'd love to do that.

I did look at the release notes and some of the PRs linked therein as well as the docs... but nothing popped out at me as "this is why the semver # changed from 5 to 6 and here is what you need to know". I also tried to join the slack but couldn't figure out how. I also tried a quick search for other similar "discussions" but didn't find any.

Thanks in advance!

**Comment by mcollina** [comment, marked as answer]
v6 dropped support for Node v14 and v16. There were no api changes.
